api/api_test.go

```diff
@@ -499,7 +499,7 @@ func TestMain(m *testing.M) {
 	// Slot 7
 	nextForgers[6].Coordinator = nonBootForger
 
-	var buckets [common.RollupConstNumBuckets]common.BucketParams
+	buckets := make([]common.BucketParams, 5)
```

Comment:
Instead of `5`, could be used the defined constant `common.RollupConstNumBuckets` (which has value `5` also)?